.github/workflows/inkomoko.yaml

```diff
@@ -0,0 +1,43 @@
+name: Deploy to ECR
+
+on:
+
+  push:
+    branches: [ inkomoko ]
+
+jobs:
+
+  deploy:
+
+    name: Deploy image to ECS
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - uses: kamiazya/setup-graphviz@v1
+
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID_moni_africa }}
```

Comment:
Add SECRET_KEY and ACCESS KEY as secret to the actions, you can change this from moni_africa to inkomoko